lib/matplotlib/collections.py

```diff
@@ -536,6 +554,42 @@ def set_linestyle(self, ls):
         self._linewidths, self._linestyles = self._bcast_lwls(
             self._us_lw, self._us_linestyles)
 
+    def set_capstyle(self, cs):
+        """
+        Set the capstyle for the collection.
+
+        ACCEPTS: ['butt' | 'round' | 'projecting']
```

Comment:
We're using numpydoc style docstring. Can you delete this line, but add this to the parameters list? (See comment bellow).

Reply:
Until we *actually* get rid of the kwdoc mechanism (whether we do so is a separate issue), I would prefer leaving the ACCEPTS strings there, as it avoids having an ugly `property -- unknown` entry in the rendered table.  (But that does not preclude documenting them in the parameters table below *too*.)